src/main/java/io/syndesis/extension/TodoReport.java

```diff
@@ -0,0 +1,48 @@
+package io.syndesis.extension;
```

Comment:
as mentioned in the mail, I would go for a more specific package name, which does not conflict with our core packages. I think, a 'io.syndesis.example.extension.tp3'  or would be cool, with a Maven group 'io.syndesis.example'.

Reply:
@rhuss sure will do :)